app/serializers/garden_serializer.rb

```diff
@@ -0,0 +1,6 @@
+class GardenSerializer
+  include FastJsonapi::ObjectSerializer
+  attributes :latitude
+  attributes :longitude
+  attributes :name
+end
```

Comment:
These can be done on one line separated by commas but it isn't a problem of any sorts.

Reply:
That is good to know for the future. Thanks.